api/src/main/java/org/openmrs/module/idgen/service/IdentifierSourceService.java

```diff
@@ -217,7 +217,7 @@
      * @throws non-unique exception if more than one auto-generation option for this type
      */
     @Transactional(readOnly = true)
-    public AutoGenerationOption getAutoGenerationOption(PatientIdentifierType type) throws APIException;

```

Comment:
why did you change this

Reply:
Same for this as well

Reply:
it shows as a change from your end

Reply:
If you didn't change them then your commits are mixed with someone else's commits

Reply:
Your commit should not have any changes to this file.